webcompat/api/uploads.py

```diff
@@ -100,6 +106,10 @@ def save(self):
             save_parameters['save_all'] = True
         # unpacking save_parameters
         self.image_object.save(file_dest, **save_parameters)
+        # Creating the thumbnail
+        size = (150, 150)
+        self.image_object.thumbnail(size, Image.ANTIALIAS)
```

Comment:
From the docs, 

> You should use PIL.Image.ANTIALIAS unless speed is much more important than quality.

http://pillow.readthedocs.io/en/3.1.x/releasenotes/2.7.0.html#default-filter-for-thumbnails suggests that `BICUBIC` is faster. 

Have you tested (or noticed) any speed differences?